nginx/proxy_ssl.conf

```diff
@@ -2,14 +2,25 @@ upstream target_service {
   server {{TARGET_SERVICE}};
 }
 
+upstream cert_service {
+  server {{CERT_SERVICE}};
```

Comment:
I've tried a few things but there is no easy way to do this in plain nginx conf without using some template language to generate the configuration...